File: src/main/java/io/github/toolsjs/kubejs/builders/HammerItemBuilder.java
package io.github.toolsjs.kubejs.builders;

import dev.latvian.mods.kubejs.item.custom.HandheldItemBuilder;
import net.minecraft.resources.ResourceLocation;
import net.minecraft.world.item.Item;
import thedarkcolour.exdeorum.item.HammerItem;


public class HammerItemBuilder extends HandheldItemBuilder {
    public HammerItemBuilder(ResourceLocation i) {
        super(i, 0.5F, 0.5F);
    }

    @Override
    public Item createObject() {
        return new HammerItem(toolTier, createItemProperties());
    }
}

File: src/main/kotlin/io/github/toolsjs/kubejs/builders/exdeorum/HammerItemBuilder.kt
package io.github.toolsjs.kubejs.builders.exdeorum

import dev.latvian.mods.kubejs.item.custom.HandheldItemBuilder
import net.minecraft.resources.ResourceLocation
import net.minecraft.world.item.Item
import thedarkcolour.exdeorum.item.HammerItem

class HammerItemBuilder(i: ResourceLocation?) : HandheldItemBuilder(i, 0.5f, 0.5f) {
    override fun createObject(): Item {
        return HammerItem(toolTier, createItemProperties())
    }
}
